cmd/backup.go

```diff
@@ -56,8 +55,6 @@ func NewBackupCommand() *cobra.Command {
 
 			// Do not run ddl worker in BR.
 			ddl.RunWorker = false
-			// Do not run stat worker in BR.
-			session.DisableStats4Test()
```

Comment:
can we disable the stats loop after we have dumped all JSONTables?

Reply:
when we reached here, we have domain already. so it seems we can not touch the `statsLease`, https://github.com/pingcap/tidb/blob/e136429d8dc5d70f43cd3f94179b0b9f47595097/session/tidb.go#L68